### ISSUE TITLE
Autocomplete doesn't list pandas.testing

### ISSUE THREAD

**Issue by drorata** [microsoft/python-language-server#1395]
VScode version: 1.36.1
ms-python.python: 2019.6.24221
Microsoft Python Language Server version 0.3.43.0

In the terminal I have a virtual environment which was set by `conda` with Python version 3.7.3. The packages in the environment were installed from a `requirements.txt` file:

```
pytest
dvc[s3]==0.51.2
click==7.0
numpy==1.16.4
pandas==0.24.2
pyarrow==0.14.0
dask[complete]==2.1.0
graphviz==0.11.1
```

In the following, 

![image](https://user-images.githubusercontent.com/2905978/62121087-9508d500-b2cb-11e9-9bcf-74a67f32aeb1.png)

I was expecting to see the suggestion for `.testing` and not only for `test`. Is it a bug or a problem in my settings?

**Comment by gramster** [comment]
This looks like a bug. Thanks for reporting it. We will investigate.

**Comment by MikhailArkhipov** [comment]
Same as #1301
![image](https://user-images.githubusercontent.com/12820357/65160848-5b12a000-d9eb-11e9-87e9-a43567539381.png)

**Comment by MikhailArkhipov** [comment]
0.4.26+

**Comment by jakebailey** [comment]
The build with the fix is 0.4.26, so the fix will be available in 0.4.26 and above. The language server is released independently from VS Code or the Python extension and this update will be pushed out to everyone once we've had time to verify there aren't any problems. If you want to test early builds before then, you can use one of the other download channels, `beta` or `daily`:

```json
"python.analysis.downloadChannel": "beta"
```

```json
"python.analysis.downloadChannel": "daily"
```

Which will get the fix sooner (not just now, we're still working on building it).

**Comment by MikhailArkhipov** [comment]
https://github.com/microsoft/python-language-server/wiki/Versioning-and-download-channels
